.github/workflows/release.yml

```diff
@@ -0,0 +1,141 @@
+name: Release Events
+
+on:
+  release:
+    types: [published]
+
+concurrency:
+  group: "${{ github.head_ref || github.ref }}"
+  cancel-in-progress: true
+
+jobs:
+  deploy-prod:
+    runs-on: ubuntu-latest
+    if: github.ref == 'refs/heads/main'
```

Comment:
Are you sure that this will work for a release trigger? I've been using:

`if: github.event.release.prerelease == false`